app/views/home/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1 class="text-center">Khan Academy Myanmar Community</h1>
-<p>This is the home page.</p>
+<div class="row">
+	<div class="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-xs-12" style="float:left;text-align:center;margin-top:100px;">
+    <img src='/khan-academy.png'>
+  </div>
+  <div class="col-x1-12 col-lg-12 col-md-12 col-sm-12 col-xs-12 " style="float:right;text-align:center;margin-top:-20px;">
+    <h1 class="text-center">Contribute to Learning Today!</h1>
+  	<h5>Join us in bringing free education to Myanmar by translating important videos from Khan Academy</h5>
+
+  	<a href="/users/sign_in" class="btn btn-default navbar-btn navbar-right" style="margin:auto;margin-top:5px;float:None !important;">Get Started</a>
```

Comment:
Ditto about the HTML styling. This also isn't a `navbar-btn` or `navbar-right`, so you should get rid of those. To center this element, put it inside a div with class `text-center`.

Also, especially since we're using rails, don't hardcode the relative paths. Instead, use embedded Ruby to generate these elements. This element can be coded as:
```<%= link_to 'Get Started', new_user_registration_path, :class => 'btn btn-default' %>```